st_package_reviewer/check/file/check_os_system_calls.py

```diff
@@ -0,0 +1,31 @@
+from . import FileChecker
+import ast
+
+class CheckOsSytemCalls(FileChecker):
+    """Checks for any calls to os.system and suggests to use subprocess.check_call intead."""
+
+    def check(self):
+        self.counter = 0
+        self.visit_all_pyfiles()
+        if self.counter > 0:
+            self.warn("Detected os.system usage: Make sure you thought about the platform key in "
```

Comment:
The current structure causes the same heuristic to raise at least two warnings. This collective report style should imo not be used for checking tools and instead every single instance of this check failing should raise the same message, as happening later in line 17.